_licenses/lal-1.3.txt

```diff
@@ -0,0 +1,214 @@
+---
+title: Licence Art Libre 1.3
+spdx-id: LAL-1.3
+
+description: This license is written in consideration of both French law and the Berne Convention for the Protection of Literary and Artistic Works. Frequently used for media assets such as logos. When incorporated into larger works, the larger work is expected to be either publicly accessable or similarly licensed.
+
+how: Create a text file (typically named LICENSE or LICENSE.txt) in the root of your source code and copy the text of the license into the file. It is also acceptable to solely supply a link to a copy of the license, usually to the <a href='https://artlibre.org/'>canonical URL for the license</a>.
+
+using:
+  - Open Source Publishing: https://github.com/osp/osp.workshop.flatland/blob/master/licences/lal.txt
+  - linuxfr.org: https://github.com/mjourdan/linuxfr-design/blob/master/LICENCE.md
+  - Oak: https://github.com/Debetux/Oak/blob/master/template/default/LICENCE
```

Comment:
OK, these are French now, but they need to be examples with the full text of the license rather than just statements that a project is under this license.

Reply:
518c4bcf7a8750dc7ebb3612b79730c4674c8f61